src/cpp/session/modules/SessionHistory.cpp

```diff
@@ -361,12 +361,20 @@ void onHistoryAdd(const std::string& command)
    module_context::enqueClientEvent(event);
 }
 
-SEXP rs_timestamp(SEXP dateSEXP)
+SEXP rs_timestamp(SEXP stampSEXP, SEXP prefixSEXP, SEXP suffixSEXP, SEXP quietSEXP)
 {
-   boost::format fmt("##------ %1% ------##");
-   std::string ts = boost::str(fmt % r::sexp::safeAsString(dateSEXP));
+   std::string prefix = r::sexp::safeAsString(prefixSEXP);
```

Comment:
I agree, as I recall an invalid `boost::format` will throw an exception which is basically a system crash in our environment.